src/gdcmodels/init_index.py

```diff
@@ -157,6 +173,33 @@ def init_index(args: Arguments):
                 settings=es_models[index][index_type].settings,
                 mappings=es_models[index][index_type].mappings,
             )
+            indices_created += 1
+
+        # Skip this step if no alias provided.
+        if not alias:
+            continue
+
+        # Skip this step if no index was created in the above steps.
+        if not es.indices.exists(index=full_index_name):
+            print(f"Index '{index}' not created so alias '{alias}' will not be created.")
+            continue
+
+        # This script uses the index names as defined by es_models and creates
+        # indices from those instead of naively creating elements from the
+        # --index flag directly. That means 'gdc_from_graph' can create
+        # multiple indices despite only being one of the elements in the
+        # --index flag. This breaks the alias logic but we don't use this
+        # script to create those indices so this conditional should never
+        # happen in practice.
+        if indices_created > 1:
+            print(f"Cannot create alias '{alias}' because too many indices created for '{index}'")
```

Comment:
`print`? Should this be logging instead?

Reply:
I typically follow conventions in the code base I'm working on but it sounds like maybe I should have ripped out any existing logging and converted it to use the python logger instead. I can make those changes.